### PR TITLE
Added "About this article" header above footer menu.

### PR DESCRIPTION
**Pull request by montehurd**
https://phabricator.wikimedia.org/T122109

screen shot 2015-12-23 at 12 52 02 am

### CODE REVIEW

File: Wikipedia/Code/WMFArticleFooterMenuViewController.m [hunk 1]
```diff
@@ -39,6 +39,16 @@ - (void)viewWillAppear:(BOOL)animated {
     [self.tableView deselectRowAtIndexPath:[self.tableView indexPathForSelectedRow] animated:YES];
 }
 
+- (CGFloat)tableView:(UITableView *)tableView heightForHeaderInSection:(NSInteger)section {
```
Comment (bgerstle) [inline Wikipedia/Code/WMFArticleFooterMenuViewController.m]: why can't we use a plain table instead? will "about this article" ever have multiple sections?

Comment (montehurd) [inline Wikipedia/Code/WMFArticleFooterMenuViewController.m]: @bgerstle 
With grouped tables we don't have to have any special logic to have separator on top of the first cell and on bottom of the last cell.

Comment (bgerstle) [inline Wikipedia/Code/WMFArticleFooterMenuViewController.m]: ah, gotchya. i knew we had discussed this before, but forgot. would this also work if we set `sectionHeaderHeight` and `sectionFooterHeight` properties in `viewDidLoad`?  might also be worth DRYing this up into a category and explaining why we do this in doc comments.

Comment (montehurd) [inline Wikipedia/Code/WMFArticleFooterMenuViewController.m]: > would this also work if we set sectionHeaderHeight and sectionFooterHeight properties in viewDidLoad

No unfortunately that doesn't seem to work...

screen shot 2015-12-23 at 11 45 54 am

**Pull request merged**